themes/default/content/docs/guides/pulumi-packages/schema.md

```diff
@@ -318,6 +318,16 @@ For `csharp`:
 | `rootNamespace`          | `string`      | No       | The root namespace that the generated package should live under. This setting defaults to "Pulumi". |
 | `respectSchemaVersion`   | `boolean`     | No       | Use the [`package.version`](#package) field in the generated SDK.                                   |
 
+For `java`:
+
+| Property       | Type          | Required | Description                                                                                      |
+|----------------|---------------|----------|--------------------------------------------------------------------------------------------------|
+| `packages`     | `map[string]` | No       | Overrides for module names to Java package names. Example: "autoscaling/v1" -> "autoscaling.v1". |
+| `basePackage`  | `string`      | No       | Prefixes the generated Java package. This setting defaults to "com.pulumi".                      |
+| `buildFiles`   | `string`      | No       | Generates build files for the chosen build tool. Currently supported values: "gradle".           |
```

Comment:
```suggestion
| `buildFiles`   | `string`      | No       | Generates build files for the chosen build tool. Supported values: "gradle".           |
```

we try to avoid coming soon, and currently language as it is misleading and difficult to maintain